client/my-sites/comments/comment-list/index.jsx

```diff
@@ -494,31 +490,6 @@ export class CommentList extends Component {
 					{ isEnabled( 'comments/management/m3-design' ) &&
```

Comment:
What do you think of also removing this flag check on both `<Comment>` and its related placeholder?

Reply:
Will start a separate PR for feature flags clean up.

Reply:
Yeah, it just feels odd that if that flag is disabled we'll render nothing here 😄 

*(also in this case I'll removing the flag anyway in #20745 😛 )*

Reply:
But it's enabled everywhere now, right?

Reply:
Removing feature flags in https://github.com/Automattic/wp-calypso/pull/20769.